### DISCUSSION TITLE
Color Blind Mode contains green

### DISCUSSION THREAD

**Discussion by ramonaerne** [community/community#52045] [Accessibility]
### Select Topic Area

Bug

### Body

Since recently bars (squares) indicating the number of changes in a file "+/-" contain green boxes instead of blue.

I can see this in the day theme of Light Protanopia & Deuteranopia  and Light Tritanopia but not in the dark theme

![image](https://user-images.githubusercontent.com/966318/230038860-98041362-dc2f-4aa3-b432-6356d2ad3084.png)

EDIT: it seems there is more buttons that have been wrongly colored

* PR CI runs
![image](https://user-images.githubusercontent.com/966318/230044569-5ac6a975-385b-470e-8717-38da884ed097.png)

* comment editing fields
 
![image](https://user-images.githubusercontent.com/966318/230044647-b4011ff6-a841-4b26-baa3-6f53394a3940.png)

**Comment by drabb** [comment]
Yes.  The **Protanopia & Deuteranopia** theme(s) seemed to have changed in the last 48 hours or so and not for the better.  The contrast and accessibility of this theme _was_ one of the better ones out there. 😞

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ramonaerne ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by cehfisher** [comment]
Hello @ramonaerne - thank you for providing this feedback! We wanted to follow up on this reported bug. We were unable to replicate this issue in our review of both the Light Protanopia & Deuteranopia and Light Tritanopia themes. We are thinking there may have been some updates to these themes after you reported the issue originally.

<img width="148" alt="Screenshot of blue and red file change squares" src="https://github.com/community/community/assets/15240149/567300c4-2385-4217-8107-86fa5691b383">


Is this an issue you can replicate currently? If the original issue is resolved, could you please close this out? If it is not resolved, could you supply more detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
